[IMP] pos: update customer display page (#17370)
odoo/documentation: content/applications/sales/point_of_sale/use.rst

Notes allow you to attach extra information to specific products in an order. There are two types of
notes: internal notes and customer notes.

.. note::

>    Any notes added to a product from the :ref:`POS register <pos/use/open-register>` will be

Reviewer: ```suggestion
   Any notes added to a product from the :ref:`POS register <pos/use/open-register>` are
```
(good thinking adding this note in this doc!)
